.cbmc-batch/jobs/aws_cryptosdk_sig_sign_start/aws_cryptosdk_sig_sign_start_harness.c

```diff
@@ -23,29 +23,30 @@
 #include <cipher_openssl.h>
 
 void aws_cryptosdk_sig_sign_start_harness() {
-    /* arguments */
+    /* Nondet Inputs */
     struct aws_cryptosdk_sig_ctx *ctx;
     struct aws_allocator *alloc = can_fail_allocator();
     struct aws_string *pub_key;
     struct aws_string *priv_key = ensure_string_is_allocated_nondet_length();
     enum aws_cryptosdk_alg_id alg_id;
     struct aws_cryptosdk_alg_properties *props = aws_cryptosdk_alg_props(alg_id);
 
-    /* assumptions */
-    __CPROVER_assume(props);
+    /* Assumptions */
+    __CPROVER_assume(props != NULL);
```

Comment:
Why can't we assume here `aws_cryptosdk_alg_properties_is_valid()`?

Reply:
Updated!